### ISSUE TITLE
Implement physics loss functions

### ISSUE THREAD

**Issue by badulion** [MAGNET4Cardiac7T/magnet-pinn#18]
Implement Divergence and Curl of 3d-field with arbitrary accuracy.

- Use findiff for calculation of filter coefficients
- Filter coefficients as pytorch conv3d
- Implement Maxwell Equations as losses composed using curl and div operators
